tests/e2e/test-data/certs/certsByPath/clientPath.crt

```diff
@@ -0,0 +1,3 @@
+-----BEGIN CERTIFICATE-----
+mockedClientCrtPath
+-----END CERTIFICATE-----
```

Comment:
Couldn't you use one file for this and the two above?
Same for others which are the same

Reply:
no, I'm testing here import of certificates by path(on machine), there are cases with the same certificates body and different certificates names and with different body and the same names

that's why we need multiple files here